tests/openvino/test_quantization.py

```diff
@@ -416,7 +456,8 @@ def test_automodel_static_quantization(
     ):
         model_id = MODEL_NAMES[model_name]
         task = model_cls.export_feature
-        dataset_name, dataset_config_name, column_name = _TASK_TO_DATASET[task]
+        dataset_kwargs = {**_TASK_TO_DATASET[task]}
```

Comment:
why exploding this dict only to re-create it ?

Reply:
This is because we need `dataset_kwargs` to contain a copy of `_TASK_TO_DATASET[task]` since we modify it below `dataset_kwargs.pop("column_name")`.